### ISSUE TITLE
Decrement font size as long as lower then size

### ISSUE THREAD

**Issue by kwladyka** [yogthos/clj-pdf#82]
I get cutom text and have to fit it into some size. I have to decrease font size as long as it will fit into this size. How can i achieve that using clj-pdf?

For example text "foo bar foo bar foo bar foo bar" has to fit in 38x20 size. So i have to check width and size for this text and if not fit decrease font size and try again.

**Comment by yogthos** [comment]
Unfortunately, there's no support for automatic resizing based on page margins at the moment. I haven't really investigated whether this would be possible.

**Comment by kwladyka** [comment]
I was using something like that many years ago in PHP so probably it should be possible also here.

I could create some element and after that check the size of this element. If wasn't like i wanted i could remove/change that element with new element.

*new element - the same but with decreased font size.

and repeat that as long as the size not pass the requirement.

So i guess it should be some option to get size of the element after create it and remove this element.

**Comment by yogthos** [comment]
Yeah that should be doable. I keep meaning to add a `:raw` tag that will allow manipulating the java objects directly. Might be a good fit for situations like this. The tag could point to a function that takes the current document and adds whatever element it needs to it.
